### PR TITLE
feat: Add Ecosystem file paths to CODEOWNERS

### PR DESCRIPTION
**Pull request by NisanthanNanthakumar**
## Objective
The goal of this PR is to add the most obvious paths to CODEOWNERS.

### CODE REVIEW

**Comment by MeredithAnya** [comment]
I think `sentry/mediators` also should be in ecosystem (at least for now)

**Comment by benvinegar** [comment]
Codeowners file appears to support wildcards (`*`), did we consider tagging `sentry_app*`?

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -126,3 +126,41 @@ Makefile                    @getsentry/owners-sentry-dev
...
+/src/sentry/shared_integrations/              @getsentry/ecosystem
+/src/sentry/mediators/                        @getsentry/ecosystem
+
+
+/src/sentry/models/integration.py             @getsentry/ecosystem
+/src/sentry/models/integrationfeature.py      @getsentry/ecosystem
+/src/sentry/models/sentryapp.py               @getsentry/ecosystem
+/src/sentry/models/sentryappavatar.py         @getsentry/ecosystem
+/src/sentry/models/sentryappcomponent.py      @getsentry/ecosystem
+/src/sentry/models/sentryappinstallation.py   @getsentry/ecosystem
```
Comment (manuzope) [inline .github/CODEOWNERS:157]: Maybe sentryapp*?

**Review by manuzope — COMMENTED** [review]
I'm not seeing much FE stuff. Is that harder to add a file path for?

**Comment by NisanthanNanthakumar** [comment]
> I'm not seeing much FE stuff. Is that harder to add a file path for?

I think everything that Ecosystem owns on the frontend is in `/src/sentry/static/sentry/app/views/organizationIntegrations`

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -126,3 +126,38 @@ Makefile                    @getsentry/owners-sentry-dev
...
+
+
+/src/sentry/models/integration.py             @getsentry/ecosystem
+/src/sentry/models/integrationfeature.py      @getsentry/ecosystem
+
+/src/sentry/tasks/integrations.py             @getsentry/ecosystem
+
+/src/sentry_plugins/                          @getsentry/ecosystem
+
+# To find matching files -> find . -name "*sentry_app*.py"
```
Comment (manuzope) [inline .github/CODEOWNERS:159]: Eh, don't think we need this comment. The next two lines are pretty clear. :)

**Review by manuzope — APPROVED** [review]

File: .github/CODEOWNERS (hunk 2 above)
Comment (joshuarli) [reply .github/CODEOWNERS:159]: I agree, it's not necessary.

**Pull request merged**